### ISSUE TITLE
Every time set to .adopted... this polyfill always generate new sets of styles but not removing the old one

### ISSUE THREAD

**Issue by Jack-Works** [calebdwilliams/construct-style-sheets#3]
![image](https://user-images.githubusercontent.com/5390719/58549254-b0108780-823d-11e9-9074-980ca1f6d59e.png)

**Comment by calebdwilliams** [comment]
@Jack-Works did your PR fix this issue or is this still outstanding?

**Comment by Jack-Works** [comment]
No, I'm not fixing it. Just remove the old styling will break the style. I think it caused by the order of stylesheets

**Comment by Jack-Works** [comment]
I think it may be a problem of https://github.com/mui-org/material-ui/issues/17126 instead of this project

**Comment by Lodin** [comment]
@Jack-Works, does the issue appear again? In #9, I fixed a couple of possible causes of the issue, so it should be solved after merge.

**Comment by Jack-Works** [comment]
I didn't check it, never mind, I think it works well now
